docs/source/conf.py

```diff
@@ -72,7 +72,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['.static']
+html_static_path = []
```

Comment:
do we not have any images or anything?

Reply:
Not there at least. We have 2 pngs in docs/tutorial_files.

```
Running Sphinx v3.1.2
making output directory... done
WARNING: html_static_path entry '.static' does not exist
```